crates/sui-json-rpc/src/metrics.rs

```diff
@@ -221,7 +222,7 @@ impl Logger for MetricsLogger {
             .observe(req_latency_secs);
 
         if let Some(code) = error_code {
-            if code == jsonrpsee::types::error::CALL_EXECUTION_FAILED_CODE {
+            if code == CALL_EXECUTION_FAILED_CODE || code == INTERNAL_ERROR_CODE {
```

Comment:
Does a similar change need to be made for transient error codes?

Reply:
Not sure if we want to record transient errors in our metrics, as they contribute to the alerts set up around server errors. @longbowlu @gegaowp what r ur thoughts

Reply:
I think we can record transient errors with a new metric